release_notes/index.adoc

```diff
@@ -20,3 +20,11 @@ technical notes, see the
 https://docs.openshift.com/enterprise/3.1/release_notes/index.html[OpenShift
 Enterprise 3.1 Release Notes].
 endif::[]
+
+[[release-versioning-policy]]
+== Versioning Policy
+{product-title} does not have non-backwards compatible changes and, therefore,
+does not drop features on versions earlier than v3. Some upgrades may require
+changes by the administrator. Clients will continue to work against major
+version servers. For example, a 3.1 client will work against 3.0, 3.1, and 3.2.
+Not all features are available in 3.1 clients against 3.2 servers.
```

Comment:
I think it is trying to express:
 1. a 3.2 server may have additional capabilities that a 3.1 client cannot use.
 2. a 3.2 client may have additional capabilities that are not supported by a 3.1 server.